src/DependencyGraph/AddressMapping/AddressMapping.ts

```diff
@@ -98,6 +98,25 @@ export class AddressMapping {
     sheetMapping.setCell(address, newVertex)
   }
 
+  public moveCell(source: SimpleCellAddress, destination: SimpleCellAddress) {
+    const sheetMapping = this.mapping.get(source.sheet)
+    if (!sheetMapping) {
+      throw Error('Sheet not initialized')
+    }
+    if (source.sheet !== destination.sheet) {
+      throw Error('Cannot move cells between sheets.')
+    }
```

Comment:
what is the rationale behind function that moves cell, but only inside the same sheet? how do we handle other moves?

Reply:
It moves array cells in address mapping after adding rows or columns and right now it has nothing to do with `moveCells` public api. Maybe I should change name of this function.